inductiva/fluids/wind_tunnel/README.md

```diff
@@ -22,22 +22,29 @@ Launching a simulation returns a task object, which can be used to verify the st
 
 ### Example:
 
-To run this example download a vehicle geometry in OBJ format from [here](/resources/geometry/test_vehicle.obj) and
-do not forget to insert your API Key (get one by filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link)).
+To run this example you need an object. You can fetch any object you like directly with Inductiva package. On this example, we download a test object saved in our Github repository.
+
+Do not forget to insert your API Key (get one by filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link)).
 
 ```python
 import inductiva
 
 inductiva.api_key = "YOUR_API_KEY"
 
+# Url to a test object in Inductiva Github repository
+object_url = "https://raw.githubusercontent.com/inductiva/inductiva/main" \
+              "/resources/test_vehicle.obj"
+test_object_path = inductiva.utils.files.download_from_url(
+    url=vehicle_url, local_file_path="test_object.obj")
```

Comment:
In this example, we can assume that we will preserve the filename, and that we don't need to explicitly use keyword arguments, just use positional, ie:

`vehicle_filepath = inductiva.utils.files.download_from_url(vehicle_url)`

Reply:
Alright! Will add that!

Reply:
thanks!